pkg/api/steve/supportconfigs/generator.go

```diff
@@ -51,7 +59,19 @@ func NewHandler(scaledContext *config.ScaledContext) Handler {
 // ServerHTTP implements http.Handler - attempts to authenticate/authorize the user. Returns a tar of support information
 // if the user can get the backing configmap in the adapter namespace
 func (h *Handler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
-	authorized, err := h.authorize(request)
+	var cspChartNamespace string
+	var cspChartName string
+
+	if _, ok := request.URL.Query()["usePAYG"]; ok {
```

Comment:
Nit: I think that it would make sense here to check that this value is set to true as well for a better UX. So something like:

```suggestion
	if usePAYG, ok := request.URL.Query()["usePAYG"]; ok && usePAYG == "true"{
```

Reply:
Will do.